File: bayramov/push.js
import { Api, JsonRpc, RpcError, JsSignatureProvider } from 'ineryjs/dist/index.js'
import * as dotenv from 'dotenv'

dotenv.config()

const rpc = new JsonRpc(process.env.URL)
const signatureProvider = new JsSignatureProvider([process.env.PRIVATE_KEY])
const api = new Api({ rpc, signatureProvider })
const contract = process.env.INERY_ACCOUNT
const actor = process.env.INERY_ACCOUNT

async function performAction(id, user, data) {
  try {
    const tx = await api.transact({
      actions: [{
        account: contract,
        name: 'create',
        authorization: [{
          actor,
          permission: 'active'
          }],
        data: { id, user, data }
        }]
      }, { broadcast: true, sign: true })
    
    console.log('Transaction successful:', tx)
    } catch (error) {
      console.error('Transaction failed:', error)
      }
  }

async function performTransfer(id) {
  try {
    const tx = await api.transact({
      actions: [{
        account: contract,
        name: 'destroy',
        authorization: [{
          actor,
          permission: 'active'
          }],
        data: { id }
        }]
      }, { broadcast: true, sign: true })
    
    console.log('Transaction successful:', tx)
    } catch (error) {
      console.error('Transaction failed:', error)
      }
  }

async function main(id, user, data) {
  await performAction(id, user, data)
  await performTransfer(id)
  }

main(1, process.env.INERY_ACCOUNT, 'CRUD Transaction via JSON RPC')

File: kenem/push.py
import json
import os
from eospy.cleos import Cleos

ce = Cleos(yoururl)#edit here

PRIVATE_KEY = os.environ.get('PRIV_KEYS') #edit here
ACTOR = os.environ.get('INERY_NAME')#edit here
ACCOUNT = 'username'#edit here

def create(id, user, data):
    payload = [
        {
            'account': ACCOUNT,
            'name': 'create',
            'authorization': [{
                'actor': ACTOR,
                'permission': 'active',
            }],
            'data': {
                'id': id,
                'user': user,
                'data': data
            },
        }
    ]
    trx = {"actions": payload}
    resp = ce.push_transaction(trx, key_strings=[PRIVATE_KEY], broadcast=True)
    print("=" * 75)
    print("=================== CREATE transaction information ====================")
    print("=" * 75)
    print(json.dumps(resp, indent=4))
    print("Response from contract :", resp['processed']['action_traces'][0]['console'])
    print()

def destroy(id):
    payload = [
        {
            'account': ACCOUNT,
            'name': 'destroy',
            'authorization': [{
                'actor': ACTOR,
                'permission': 'active',
            }],
            'data': {
                'id': id
            },
        }
    ]
    trx = {"actions": payload}
    resp = ce.push_transaction(trx, key_strings=[PRIVATE_KEY], broadcast=True)
    print("=" * 75)
    print("================== DESTROY transaction information ====================")
    print("=" * 75)
    print(json.dumps(resp, indent=4))
    print("Response from contract :", resp['processed']['action_traces'][0]['console'])
    print()

def main(id, user, data):
    create(id, user, data)
    destroy(id)

if __name__ == '__main__':
    main(1, ACCOUNT, "Mess")#edit here
